### ISSUE TITLE
Transparency is not reflected in the object I created in blender.

### ISSUE THREAD

**Issue by miracle777** [KhronosGroup/glTF-Blender-IO#1695]
I modeled it in blender.
Only the diamond object has no material applied.

I have installed glTF shader.

I set the blend mode to alpha blend.

The relevant images can be found on the page where I posted the question.
https://teratail.com/questions/cn0k9u9atobcpn

If I had a texture of the diamond, I could test it, but with the texture created by blender's UV map, the shape of the diamond was lost.
Therefore, it would be helpful if you could solve this problem without connecting the texture to the alpha.

**Comment by miracle777** [comment]
I used glTF TOOL and solved the problem.

I used glTF TOOL to import the GLB file and check the contents.

It was not there at first.
But in blender, I modified the alpha value in the base color section and exported it, and was able to find the correct spot to modify it.

I was able to find the baseColorFactor entry in the Materials section.
I noticed that these values are the RGB and alpha values.

If I set the initial value of 1 to 0, I get an error.
So I set it to 0.5 and was able to set the transparency.
